tests/tools/test_dist_part.py

```diff
@@ -1,5 +1,6 @@
 import json
 import os
+import sys
```

Comment:
split the changes for distributed partition into another separate PR.

Reply:
I added a new file test_dist_partition_graphbolt to check it